src/services/logger/formatters.ts

```diff
@@ -0,0 +1,96 @@
+import * as Hapi from 'hapi';
+import * as Cookie from 'cookie';
+import * as JWT from 'jsonwebtoken';
+import { compose } from 'ramda';
+import { RequestErrorEvent, RequestSentEvent, LogLine } from './types';
+import { getConfig } from '../../../config';
+
+
+const { auth: { standard } } = getConfig(process.env.NODE_ENV);
+
+
+const tagOrder: (keyof LogLine)[] = [
+  'event',
+  'method',
+  'path',
+  'statusCode',
+  'origin',
+  'userId',
+  'organisationId',
+  'query',
+  'payload',
+  'response',
+];
+
+const errorOrder: (keyof LogLine)[] = ['errorMessage', 'errorStack'];
+
+const extractCookie = (c: string) => {
+  const { [standard.cookie.name]: token } = Cookie.parse(c);
+  return token;
+};
+
+const decodeToken = (c: string): { userId?: number, organisationId?: number } => {
+  let decoded: { userId?: number, organisationId?: number };
```

Comment:
Do we have a type for cookie that we can use here instead?

Also is there a reason are these values optional?

Reply:
Answered here: https://github.com/TwinePlatform/twine-api/pull/271#discussion_r239504990

Reply:
Yes, can use that instead.

Second question answered here: https://github.com/TwinePlatform/twine-api/pull/271#discussion_r239504990